### ISSUE TITLE
Preserving presentation from book to slides

### ISSUE THREAD

**Issue by siwelwerd** [TeamBasedInquiryLearning/linear-algebra#167]
See https://teambasedinquirylearning.github.io/linear-algebra/slides.html#/10/2 , among others, for example.

I tried rebuilding the slides with the instructions in the Readme, but this did not fix the issue.

**Comment by StevenClontz** [comment]
The `image@src` attribute is empty in the built slides:

https://github.com/TeamBasedInquiryLearning/linear-algebra/blob/main/docs/slides.html#L965

It seems my book-to-slides conversion produces `latex-image` elements, but what I really want is for it to analyze the book and determine the filename of the diagram the book produces when it's built so I can point to that.

Discussion: https://groups.google.com/g/pretext-support/c/qx8N4PTeW08

**Comment by sharonakrinsky** [comment]
I just came here to make the same comment. None of the images are rendering on the slides.

**Comment by siwelwerd** [comment]
@sharonakrinsky: The workaround I am using is to pull up the HTML version in another tab, and just switch over there to show the image.  But yeah, we need to get this fixed.

**Comment by StevenClontz** [comment]
I did a quick and dirty fix - let me know how badly I broke things (but I see an image at https://teambasedinquirylearning.github.io/linear-algebra/slides.html#/8/1 at least).

**Comment by siwelwerd** [comment]
Looks okay -- there are some minor scaling issues on some of them that I will think will have to be solved individually.

This may be a separate issue that I just noticed, or it may have been caused by your change -- there seem to be extra line breaks on multi-part activities now, e.g. https://teambasedinquirylearning.github.io/linear-algebra/slides.html#/23/4

**Comment by StevenClontz** [comment]
Yep, still work to do here, but (until you tell me otherwise) nothing breaking in the next week or so. Expecting to tackle these sorts of things in May; drop notes on all other presentation errata here in the meantime.

**Comment by siwelwerd** [comment]
The extra line breaks are pretty ugly in some cases, and might have a systemic fix, so should be a higher priority. Image scaling issues can wait.

Also making a note that there is something weird going on on Slide 2.5.2 https://teambasedinquirylearning.github.io/linear-algebra/slides.html#/10/3 . I just haven't had a chance to dig into the code yet to fix it.

**Comment by sharonakrinsky** [comment]
I hit 2.5.2 today and yeah, unreadable - had to switch to the book.

**Comment by siwelwerd** [comment]
Found it -- there was a missing `<p>...</p>` inside the `<statement>`

Looks better now: https://teambasedinquirylearning.github.io/linear-algebra/slides.html#/10/3

@StevenClontz Is there an easy way to validate things like this?

**Comment by sharonakrinsky** [comment]
Thank you! I need it again in about 45 minutes :)

**Comment by StevenClontz** [comment]
@siwelwerd https://github.com/PreTeXtBook/pretext-cli/issues/66 is on our TODOs

**Comment by sharonakrinsky** [comment]
Issue with 3.1.3 - not presenting correctly on slides.

**Comment by siwelwerd** [comment]
Looks okay for me in Chrome... what browser are you using?

<img width="950" alt="Screenshot 2021-03-17 164635" src="https://user-images.githubusercontent.com/30658947/111543070-8a20e380-8740-11eb-88ab-a56e2e996dfa.png">

**Comment by sharonakrinsky** [comment]
Chrome.

**Comment by sharonakrinsky** [comment]
OK - despite navigating away and back several times it didn’t work, but then refreshed the tab and it did. Weird!!!!

**Comment by siwelwerd** [comment]
Maybe some weird caching thing, I don't know! Keep letting us know as you find things. I am a few sections ahead of you (started A3 today), so I'm trying to go ahead and fix anything I see.

**Comment by siwelwerd** [comment]
The figure numbering in the slides does not seem to match the figure numbering in the HTML.

**Comment by StevenClontz** [comment]
Can you drop links for where you noticed this?

**Comment by siwelwerd** [comment]
https://teambasedinquirylearning.github.io/linear-algebra/slides.html#/27/2

**Comment by StevenClontz** [comment]
Reporting back that figure numbering is an upstream issue with rbeezer/mathbook - we're just inserting the PreTeXt figures as follows (which are automatically numbered):

```
        <figure>
          <image source="images/image-11.svg"/>
          <caption>An injective transformation and a non-injective transformation</caption>
        </figure>
```

Opening discussion at https://groups.google.com/g/pretext-dev/c/tEZRvruigP0 (really a much broader discussion, but it'd replace our current need for a conversion from pretext/book to pretext/slideshow)

**Comment by siwelwerd** [comment]
Closed by #255
